ml-agents-envs/mlagents/envs/brain_conversion_utils.py

```diff
@@ -0,0 +1,66 @@
+from mlagents.envs.brain import BrainInfo, BrainParameters, CameraResolution
+from mlagents.envs.base_env import BatchedStepResult, AgentGroupSpec
+import numpy as np
+from typing import List, Any
+
+
+def step_result_to_brain_info(
```

Comment:
This is the equivalent of BrainInfo.from_agent_proto(), right? If I'm understanding this correctly, you've dropped the NaN checks on vector obs and rewards that were in the old code.

I think we should consider keeping them. But if you want to drop them, we should at least keep that in mind when benchmarking (since they're a non-trivial for large amounts of observations).

Reply:
The checks are in rpc_utils.py (since this is where the protos are converted to BatchedStepResult)

Reply:
Ah, sorry, I didn't scroll all the way down.